docx-core/src/documents/elements/paragraph_property.rs

```diff
@@ -37,6 +38,7 @@ pub struct ParagraphProperty {
     pub(crate) div_id: Option<String>,
     #[serde(skip_serializing_if = "Option::is_none")]
     pub paragraph_property_change: Option<ParagraphPropertyChange>,
+    pub borders: Option<ParagraphBorders>,
```

Comment:
Could you please add `#[serde(skip_serializing_if = "Option::is_none")]`

```suggestion
    #[serde(skip_serializing_if = "Option::is_none")]
    pub borders: Option<ParagraphBorders>,
```